core/src/worker/workflow/mod.rs

```diff
@@ -1363,17 +1363,19 @@ fn prepare_to_ship_activation(wfa: &mut WorkflowActivation) {
         }
         fn variant_ordinal(v: &workflow_activation_job::Variant) -> u8 {
             match v {
-                workflow_activation_job::Variant::NotifyHasPatch(_) => 1,
-                workflow_activation_job::Variant::SignalWorkflow(_) => 2,
-                workflow_activation_job::Variant::DoUpdate(_) => 2,
+                workflow_activation_job::Variant::NotifyHasPatch(_) => 0,
+                workflow_activation_job::Variant::InitializeWorkflow(_) => 1,
```

Comment:
I was under the impression the contract now was that this should always be the first job on the activation, so I wonder if this should be 0 and notify has patch should be 1

Reply:
Yeah, I suppose there's really no reason not to do that. Will change.